week-5/A-dom-manipulation/exercise.js

```diff
@@ -4,30 +4,55 @@ Task 1
 
 Write JavaScript below that changes the background colour of the page when the 'Change colour' button is clicked.
 */
+var button = document.querySelector("#bgrChangeBtn");
 
-
-
+button.addEventListener("click",changeColour);
+function changeColour(){
+    document.body.style.backgroundColor = "blue";;
+}
+//button.style.backgroundColor = "green";
 /*
 Task 2
 ======
 
 When a user clicks the 'ALERT' button, an alert box should pop up with the text "Thanks for visiting Bikes for Refugees!"
 */
+var mybutton = document.querySelector("#alertBtn");
+mybutton.addEventListener("click",alerting)
 
-
-
+function alerting(){
+alert ("Thanks for visiting Bikes for Refugees!");
+}
```

Comment:
Nice, it works!  

`mybutton` would be better as `alertButton`, and the `alerting` function body has no indentation.